### DISCUSSION TITLE
Please bring back Grid view

### DISCUSSION THREAD

**Discussion by kushcfc** [netbox-community/netbox#16260] [Ideas]
I don't have a screenshot but there used to be a Grid view when you click on Tenant and the Related Objects were displayed  [RACKS, Devices, Circuits etc] in a 3X3 or 4X4 (can't remember) grid in version 3.X.X. Can we please bring that back

**Comment by kushcfc** [comment]
![image](https://github.com/netbox-community/netbox/assets/59706001/e91ccc2f-936c-4047-94fd-f5d7f56d01d1)
Found a Screenshot of what I was referring to.
The grid view under 'Stats'

**Reply by candlerb** [reply]
Between which two versions are you comparing?

In Netbox 4, e.g. https://demo.netbox.dev/dcim/sites/23/ it is rendered like this:

![image](https://github.com/netbox-community/netbox/assets/44789/93f5441d-1a3b-45be-985b-0b8ddf063019)

And I've just checked that Netbox 3.7 is the same.

ISTM that all the info you need is there, just rendered in a slightly different way (one which makes it easier to add additional stats without using up so much screen space)

Since your version shows "AS Number" as an attribute of Site, I think it must be fairly old. The new ASN model was added in [v3.1.0](https://github.com/netbox-community/netbox/releases/v3.1.0) and the old field was removed from the Site model in [v3.2.0](https://github.com/netbox-community/netbox/releases/v3.2.0) (released April 2022).

**Reply by kushcfc** [reply]
Correct, its a old version, and yes all the information is still there, I was actually wondering if we can give the option to chose which view user wants Grid or List, Grid view offers a nice overview at a glance (look and feel).

**Comment by LHBL2003** [comment]
Oh yes, I'm in favour of that too. I'm usually in favour of lists, but the tile view was a thousand times better at this point.
The example picture is very old and in the 3.x the elements were blue, you could select the desired one faster in the grid view than in the list. But I'm not always a fan of grid views :D
In other words, I've missed the grid view ever since it was removed and have been wishing for it back ever since.

**Comment by Netboxnoob** [comment]
PLEASE can we get this feature back. I really miss that view

**Reply by Netboxnoob** [reply]
Any way this can be implemented. PLEASE!!
